CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contributing Guidelines
-* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
+* You can join our [ AnitaB.org Open Source Zulip Channel](https://anitab-org.zulipchat.com/).  Each active repo has its own channel to direct questions to (for example [#VMS](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms), [#powerup](https://anitab-org.zulipchat.com/#narrow/stream/222537-powerup) or [#portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal)).  
```

Comment:
Please remove unnecessary changes from this file.

Reply:
@snitin315 As I said these changes have been merged. These were present in [#964](https://github.com/anitab-org/vms/pull/964).
I don't know how to "undo" it :sweat_smile:

Reply:
This may have happened if you branched to IsVisibleFunction directly from "Harirai-Contributing". It wouldn't happen if you had branched from your develop (unless you already merged Harirai-Contributing to your develop)

Reply:
try rebasing the commits